REPOSITORY_STANDARDS.md

```diff
@@ -0,0 +1,89 @@
+# Repository standards
+
+Here is a list of repository standards we have for our repositories. Much of the below is automated by our `dp-cli` tool for creating repositories but is documented here for completeness.
+
+## Contents
+
+- [naming conventions](#naming-conventions)
+- [visibility](#visibility)
+- [repository documentation](#repository-documentation)
+- [permissions](#permissions)
+
+## Naming conventions
+
+Repositories should be named with the prefix `dis-` to distinguish them as belonging to `Dissemination`. This helps to identify our repositories amongst the many that are part of ONSDigital.
+
+Historical repositories have the prefix `dp-` as they belonged to `Digital Publishing`.
+
+## Visibility
+
+We aim to work in the open where possible and so the majority of our repositories should be `public`.
+
+Repositories should be marked as `private` if they reveal details of:
+
+- ONS infrastructure
+- ONS security mitigations
+
+### GitHub Copilot
+
+If a repo is `private` then GitHub Copilot and similar tools should be excluded from reading the content in the repository.
+
+## Repository documentation
+
+### CODEOWNERS
+
+Every repository should have a CODEOWNERS file in it's `.github` folder. This should show the responsible team(s) for that repository.
+
+### CONTRIBUTING
+
+Every repository should have a CONTRIBUTING file in the root directory. This should make clear to engineers how they can contribute to that repository, including:
+
+- branching strategy
+- commit approach
+- merging strategy
+
+### LICENSE
+
+Every repository should have a LICENSE file in the root directory. As standard we use the MIT licencse.
+
+### PULL_REQUEST_TEMPLATE
+
+Every repository should have a PULL_REQUEST_TEMPLATE file in it's `.github` folder. This should make clear how pull requests are documented for that repository.
+
+### README
+
+Every repository should have a README file in the root directory. This should include:
+
+- how to run the service locally, if applicable
+- details of environmental configuration
+
+## Permissions
+
+Below describes the standard approach for permissions in our repositories.
+
+### Collaborators and Teams
+
+Our standard approach is as follows:
+
+| Team                     | Permission |
+|--------------------------|------------|
+| Dissemination            | write      |
+| Dissemination Tech Leads | admin      |
+| Responsible teams        | maintain   |
+| Security                 | read       |
```

Comment:
It might be worth clarifying that the security managers have been added as read only to all organisational repos at the org level (second tab in the Teams and Collaborators list screen) and therefore they do not need to be added explicitly to a repo.

Many of our repos will still have the security team added explicitly (first tab), but this is because those permissions predate the new org level access and is now redundant.

Because of this, I would actually suggest we remove the Security team from this list so as not to confuse people.